formats/filtered/resources/ts/Filtered/View/MapView.ts

```diff
@@ -183,7 +183,7 @@ export class MapView extends View {
 
 			// TODO: Limit zoom values to map max zoom
 
-			that.map = L.map( that.getTargetElement().get( 0 ), mapOptions );
+			that.map = L.map( <HTMLElement> that.getTargetElement().get( 0 ), mapOptions );
```

Comment:
`<HTMLElement>`??? Is this valid JS?

Reply:
It is a Typescript type assertion.
See https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions